modules/microshift-configuring-hosts-for-mirror.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-configuring-hosts-for-mirror_{context}"]
+= Configuring hosts for mirror registry access
+
+To configure a {microshift-short} host to use a mirror registry, you must give the {microshift-short} host access to the registry by creating a configuration file that maps the Red Hat registry host names to the mirror.
+
+.Prerequisites
+* Your mirror host has access to the internet.
+* The mirror host can access the mirror registry.
+* You configured the mirror registry for use in your restricted network.
+* You downloaded the pull secret and modified it to include authentication to your mirror repository.
+
+.Procedure
+. Log into your {microshift-short} host.
+
+. Enable the SSL certificate trust on any host accessing the mirror registry by completing the following steps:
+
+.. Copy the `rootCA.pem` file from the mirror registry to the MicroShift host at the `/etc/pki/ca-trust/source/anchors` directory.
```

Comment:
Could this step be a little more explicit about the location of rootCA.pem. I found it on <registry_path>/quay-rootCA.